JobBars/Jobs/DRK.cs

```diff
@@ -82,7 +82,38 @@ public static class DRK {
             })
         };
 
-        public static IconReplacer[] Icons => Array.Empty<IconReplacer>();
+        public static IconReplacer[] Icons => new[] {
+            new IconBuffReplacer(AtkHelper.Localize(BuffIds.Rampart), new IconBuffProps {
```

Comment:
Role actions like Rampart should have a unique name like this: https://github.com/0ceal0t/JobBars/blob/main/JobBars/Jobs/DRK.cs#L65

Reply:
Thanks, added job suffixes to rampart and arm's length